### PR TITLE
Feature/kc remove all subviews

### PR DESCRIPTION
**Pull request by KaneCheshire**
- Adds ability to `removeAllSubviews` and `replaceSubviews` with a new array of subviews.
- Adds unit tests
- Bumps version to `2.1.0`

### CODE REVIEW

File: TABScrollingContentViewTests/TABScrollingContentViewTests.swift [hunk 1]
```diff
@@ -0,0 +1,49 @@
+//
```
Comment (lucianomarisi) [inline TABScrollingContentViewTests/TABScrollingContentViewTests.swift]: we should probably put the test files in a `Tests` folder to follow the Swift Package Manager convention as it looks like this project already follows it as I'm seeing the `Sources` folder

File: TABScrollingContentViewTests/TABScrollingContentViewTests.swift [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+//  Created by Kane Cheshire on 20/02/2017.
+//  Copyright © 2017 The App Business. All rights reserved.
+//
+
+import XCTest
+@testable import TABScrollingContentView
+
+class TABScrollingContentViewTests: XCTestCase {
+  
+  var scrollingContentView: VerticalScrollingContentView?
```
Comment (lucianomarisi) [inline TABScrollingContentViewTests/TABScrollingContentViewTests.swift]: I'd go with an implicitly unwrapped option for your system under test, it'll clean up all the optionals in the test

File: TABScrollingContentViewTests/TABScrollingContentViewTests.swift [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  }
+  
+  func test_replacingAllSubviews() {
+    XCTAssertEqual(scrollingContentView?.contentView.subviews.count, 0)
+    let views = createViews()
+    scrollingContentView?.add(subviews: views)
+    XCTAssertEqual(views.count, scrollingContentView?.contentView.subviews.count)
+    let newViews = createViews(2)
+    scrollingContentView?.replaceSubviews(with: newViews)
+    XCTAssertEqual(scrollingContentView?.contentView.subviews.count, newViews.count)
```
Comment (lucianomarisi) [inline TABScrollingContentViewTests/TABScrollingContentViewTests.swift]: would checking for equality on views be more correct? i.e. `   XCTAssertEqual(scrollingContentView.contentView.subviews, newViews)`

File: TABScrollingContentView.xcodeproj/xcshareddata/xcschemes/TABScrollingContentViewTests.xcscheme [hunk 4]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lucianomarisi) [inline TABScrollingContentView.xcodeproj/xcshareddata/xcschemes/TABScrollingContentViewTests.xcscheme]: I don't think we need to share this scheme.

**Review by lucianomarisi — CHANGES_REQUESTED** [review]
some minor comments

File: TABScrollingContentViewTests/TABScrollingContentViewTests.swift (hunk 1 above)
Comment (KaneCheshire) [reply TABScrollingContentViewTests/TABScrollingContentViewTests.swift]: Updated

File: TABScrollingContentViewTests/TABScrollingContentViewTests.swift (hunk 2 above)
Comment (KaneCheshire) [reply TABScrollingContentViewTests/TABScrollingContentViewTests.swift]: Updated

File: TABScrollingContentView.xcodeproj/xcshareddata/xcschemes/TABScrollingContentViewTests.xcscheme (hunk 4 above)
Comment (KaneCheshire) [reply TABScrollingContentView.xcodeproj/xcshareddata/xcschemes/TABScrollingContentViewTests.xcscheme]: Updated

**Comment by lucianomarisi** [comment]
may be worth updating the project settings as I get a warning

File: TABScrollingContentViewTests/TABScrollingContentViewTests.swift (hunk 3 above)
Comment (KaneCheshire) [reply TABScrollingContentViewTests/TABScrollingContentViewTests.swift]: Makes sense, updated

**Comment by KaneCheshire** [comment]
Fixed the warning

**Comment by lucianomarisi** [comment]
could we update the `CHANGELOG.md` mentioning this:
- Adds ability to removeAllSubviews and replaceSubviews with a new array of subviews.?

**Comment by KaneCheshire** [comment]
Updated the changelog, totally forgot about that sorry :/

**Review by lucianomarisi — APPROVED** [review]

**Review by samdods — APPROVED** [review]

**Pull request merged**